src/test/java/gale/ParserTest.java

```diff
@@ -0,0 +1,36 @@
+package gale;
+import org.junit.jupiter.api.Test;
+import java.time.format.DateTimeParseException;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ParserTest {
+
+    @Test
+    public void taskCommand_wrongCommandFormat_exceptionThrown() {
```

Comment:
You got the naming right, but I'm not sure whether the name here is very intuitive! It seems like you're trying to test your parser, so I'm a bit confused on what `taskCommand` is.

Reply:
naming is not very clear, a bit confusing